### ISSUE TITLE
Shopping Cart shows cheapest price

### ISSUE THREAD

**Issue by ewinck** [organicinternet/magento-configurable-simple#13]
When an SCP configurable product is added to the cart, the price is shown as the cheapest option, even though the prices are correctly displayed on the product page. In addition, the correct options are show (ie Color - blue; Size - large), but the pricing is for the wrong product.

When I look at the SKU in the cart, it's the SKU for the configurable product. I was expecting to see the SKU for the simple product.

**Comment by toasty** [comment]
You're right - you should see the SKU for the simple product in the cart.

It's likely that your html is sufficiently different to the standard themes that SCP can't find the main product add to cart form to update it. Can you check your form tag has ID product_addtocart_form please?

Also install firebug and look to see if any javascript errors are thrown at any point, and let me know what they are.

Or just mail me a link me to a page where the problem occurs and I'll take a look whenever I get a free minute.

**Comment by ewinck** [comment]
Thanks for your help. The form does have the id product_addtocart_form, but we have seriously modified the product display page. Can you point me to what should be updating?

When I submit the 'add to cart' form, it's passing the usual variables, as far as I can tell:
Array
(
    [uenc] => aHR0cDovL2Rldi5uZWJvd2ViLmNvbS9wbGF5L2Zyb2xpYy5odG1s
    [product] => 167
    [related_product] => 
    [super_attribute] => Array
        (
            [118] => 24
            [80] => 46
        )

```
[qty] => 1
```

)

Should that be submitting the product_id of the simple product?

If it's more helpful to look at the page itself, here's a link: http://dev.neboweb.com/play/frolic.html

**Comment by ewinck** [comment]
I think I've figured out a work-around to use javascript to update the action of the form based on which options are selected. Is that how the module is supposed to add simple products to the cart? 

Thanks for pointing me in the right direction.

**Comment by toasty** [comment]
Yeah that's how it works. It's not mega elegant, as it actually regexes the end of the submit url to change which product will be added to cart. :}
SCP normally passes an extra var too, 'cpid' (configurable parent product id) so that in the cart it can link back to the correct configurable product.
